environment/docker/pool/client.ubuntu.dockerfile

```diff
@@ -5,14 +5,20 @@ ARG nodecnt
 ARG clicnt=10
 
 EXPOSE 5000-9799
-USER indy
+USER root
```

Comment:
I guess `root` user will lead to root:root ownership for `/etc/indy/indy_config.py`. I think it makes sense to use `root` only for `apt` and `pip` related installations below.

Reply:
@ozheregelya or @andkononykhin  or @VladimirWork, will you be able to process this findings so that we can merge the PR?

Reply:
@ashcherbakov I will